### PR TITLE
Feature/com cost

### PR DESCRIPTION
**Pull request by jmcarp**
Includes #1131, with election filtering, unit tests, and resolved merge conflicts.

### CODE REVIEW

File: data/sql_updates/create_communication_cost_view.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+drop materialized view if exists ofec_aggregate_communication_cost_candidate_mv_tmp;
+create materialized view ofec_aggregate_communication_cost_candidate_mv_tmp as
+select
+    org_id as cmte_id,
+    s_o_cand_id as cand_id,
+    s_o_ind as support_oppose_indicator,
+    sum(communication_cost) as total,
+    count(communication_cost) as count,
+    cast(extract(YEAR from communication_dt) AS integer) +
```
Comment (arowla) [inline data/sql_updates/create_communication_cost_view.sql]: A shorter way:
`date_part('year', communication_dt) + date_part('year', communication_dt) % 2 as cycle`

Comment (jmcarp) [inline data/sql_updates/create_communication_cost_view.sql]: :+1:

**Comment by arowla** [comment]
Aside from the minor line note, LGTM.

**Comment by jmcarp** [comment]
Thanks for reviewing!

**Pull request merged**